testonly/hammer/hammer.go

```diff
@@ -370,8 +375,14 @@ type readWorker struct {
 }
 
 func newReadWorker(s *hammerState, idx int) *readWorker {
+	readBias := MapBias{
+		Bias:          make(map[MapEntrypointName]int),
+		InvalidChance: make(map[MapEntrypointName]int),
+	}
+	// TODO(mhutchinson): populate readBias by iterating over roMapEntrypoints
```

Comment:
This seems like 3 lines of code, so maybe do it in this PR?

Reply:
No, because it changes behaviour in such a way that it causes races on the data structures, and we will start querying for non-existent revisions of the map, or finding revisions of the map which we haven't stored the local copy of yet.

For reference, I did first have it in the branch this came from, but had to separate this functional change out to identify the cause of the new failures.